### DISCUSSION TITLE
Download Jira attachments?

### DISCUSSION THREAD

**Discussion by MattPD** [ankitpokhrel/jira-cli#334] [Ideas]
Is there a way to download files attached to Jira issues? 
As in, e.g., https://confluence.atlassian.com/jiracoreserver/attaching-files-and-screenshots-to-issues-939937892.html

**Comment by ankitpokhrel** [comment]
Hi @MattPD, this feature is not implemented yet. I will see what I can do here. PRs are welcome in the meantime :)

**Reply by MattPD** [reply]
Thanks!

**Reply by wallind** [reply]
@ankitpokhrel I put up https://github.com/ankitpokhrel/jira-cli/pull/986 for you to consider.

*_see also earlier related attempt https://github.com/ankitpokhrel/jira-cli/pull/954, though obviously I prefer mine 🖤_

**Comment by oryan-block** [comment]
Any plans to support this? Would be nice if [attachment] links would work like other urls or if there would be a list of these links in an attachment section below the description.
